hi-ml-histopathology/testhisto/testhisto/utils/test_metrics_utils.py

```diff
@@ -88,7 +88,27 @@ def assert_equal_lists(pred: List, expected: List) -> None:
                  [Tensor([200, 424, 200, 424]),
                   Tensor([200, 200, 424, 424]),
                   Tensor([200, 200, 424, 424]),
-                  Tensor([200, 200, 424, 424])]
+                  Tensor([200, 200, 424, 424])],
+             ResultsKey.TILE_LEFT:
```

Comment:
Can't we now get rid of the previous TILE_X and TILE_Y results in this test?

Reply:
Thanks, done